sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/_client.py

```diff
@@ -698,7 +736,7 @@ def _start(self) -> None:
             self._is_stopping = False
             raise e
 
-    def _stop(self) -> None:
+    def stop(self) -> None:
```

Comment:
I prefer `close` as a name - that aligns with both our HTTP and AMQP clients.

Reply:
updated